forest-node/README.md

```diff
@@ -0,0 +1,35 @@
+
+## Running Forest In DigitalOcean
+
+## Forest IAC Architectue
+
+![Forest Cloud Infrastructure ](https://user-images.githubusercontent.com/47984109/216006502-eca661d3-2ef8-4c75-aa7a-1740c25abb44.png)
+
+## Requirements 
+- RAM: 32GB
+- VCPU: 8
```

Comment:
16GB with 2 cores should suffice but I guess this is something we can hone later.

Reply:
Yes sure

Reply:
Let's update the README to match the specs of the droplet we're using by default.

Reply:
Sure - done.